docs/admin/config-guide/agama-configuration.md

```diff
@@ -190,6 +190,121 @@ Also, You can deploy agama project in Janssen through commandline.
 /opt/jans/jans-cli/config-cli.py --operation-id post-agama-prj --endpoint-args name:agama-project-name
 ```
 
+**Example:**
+
+Let's upload [a test project](https://github.com/JanssenProject/jans/blob/main/docs/assets/agama/test-project.zip)
```

Comment:
Hi @devrimyatar this PR doesn't have a `test-project.zip`. Did you mean `journey.zip`? 

Also, use the relative URLs in the docs when possible. For example `../../assets/agama/test-project.zip`